docs/content/direct/packaging.md

```diff
@@ -207,12 +212,15 @@ it from local sources. This is not included in `make all-generated`.
 This chart creates (among other things) a `Deployment` object that runs a container from the [KubeStellar controller-manager container image](#kubestellar-controller-manager-container-image).
 
 The chart is published at the OCI repository
-`ghcr.io/kubestellar/kubestellar/kubestellar-operator-chart`. A GitHub Actions workflow (in `.github/workflows/goreleaser.yml`) specializes and publishes this chart as part of [the release process](release.md).
+`ghcr.io/kubestellar/kubestellar/controller-manager-chart`. A GitHub Actions workflow (in `.github/workflows/goreleaser.yml`) specializes and publishes this chart as part of [the release process](release.md).
 
 The following versions exist.
 
 | OCI tag = version in chart | appVersion in chart | referenced container tag |
 | -------------------------- | ------------------- | ------------------------ |
+| 0.22.0-rc1                 | 0.22.0-rc1          | 0.22.0-rc1 |
+| 0.21.2                     | 0.21.2              | 0.21.2 |
+| 0.21.0                     | 0.21.0              | 0.21.0 |
 | 0.20.0                     | 0.20.0              | 0.20.0 |
```

Comment:
Same as the SHA and  commit comment above - what is the purpose of this table ?  I suggest we add such a table when we start having differences between the columns. I don't see any usage of this info